### PR TITLE
Add PageTransporter & WebElementActions Task

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/assets/DeleteConfirmationPage.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * Represents page of deleting confirmation of an asset.
+ */
+public class DeleteConfirmationPage extends BasePage {
+
+    @FindBy(xpath = "//div[contains(@class,\'modal-container\')]//button[contains(@class,\'default\')]")
+    private WebElement deleteConfirmationBtn;
+
+
```
Comment (edsonarios) [inline src/main/java/salesforce/ui/pages/assets/DeleteConfirmationPage.java]: This line should be erase

File: src/main/java/salesforce/ui/pages/assets/OptionMenuPage.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Represents page of option menu in created asset.
+ */
+public class OptionMenuPage extends BasePage {
+
+    @FindBy(xpath = "//div[@role=\'menu\']//a[./div[@title=\'Eliminar\']]")
+    private WebElement deleteBtn;
+
+
```
Comment (edsonarios) [inline src/main/java/salesforce/ui/pages/assets/OptionMenuPage.java]: This line should be erase

File: src/test/java/salesforce/login/LoginTest.java [hunk 3]
```diff
@@ -5,16 +5,18 @@
...
-
+    protected CreatedAssetPage createdAssetPage;
     @BeforeClass
     public void login() {
         HomePage homePage = loginPage.loginSuccessful(EnvConfig.getInstance().getUsername(),
                 EnvConfig.getInstance().getPassword());
         assetPage = pageTransporter.navigateToAssetPage();
     }
 
+
```
Comment (edsonarios) [inline src/test/java/salesforce/login/LoginTest.java]: This line should be erase

File: src/test/java/salesforce/assets/CreateAssetTest.java [hunk 4]
```diff
@@ -9,15 +9,32 @@
...
+                "Asset was not created");
+    }
+
+    @AfterClass
+    public void deleteAsset() {
+        OptionMenuPage optionMenuPage = createdAssetPage.clickCreatedAssetOptionBtn();
+        DeleteConfirmationPage deleteConfirmationPage = optionMenuPage.clickDeleteBtn();
+        deleteConfirmationPage.clickDeleteConfirmationBtn();
+        pageTransporter.navigateToAssetPage();
     }
```
Comment (cristianfchq) [inline src/test/java/salesforce/assets/CreateAssetTest.java]: this line should be removed

**Review by cristianfchq — APPROVED** [review]

File: src/main/java/salesforce/ui/pages/assets/CreateAssetPage.java [hunk 5]
```diff
@@ -51,7 +51,25 @@
...
+    @FindBy(xpath = "(//div[@role=\'listbox\']//a)[1]")
+    private WebElement roleFirstOptionBox;
+
+    @FindBy(xpath = "(//div[@class=\'autocompleteWrapper slds-grow\']/input)[2]")
+    private WebElement productOptionBox;
+
+    @FindBy(xpath = "(//div[@class=\'listContent\']//a)[3]")
+    private WebElement productFirstOptionBox;
+
+    @FindBy(xpath = "(//div[@class=\'autocompleteWrapper slds-grow\']/input)[3]")
```
Comment (Raymundogs) [inline src/main/java/salesforce/ui/pages/assets/CreateAssetPage.java:66]: You should find locators that do not depend on the position.

File: src/main/java/salesforce/ui/pages/assets/DeleteConfirmationPage.java (hunk 1 above)
Comment (JRojas999) [reply src/main/java/salesforce/ui/pages/assets/DeleteConfirmationPage.java]: It's already done.

File: src/main/java/salesforce/ui/pages/assets/OptionMenuPage.java (hunk 2 above)
Comment (JRojas999) [reply src/main/java/salesforce/ui/pages/assets/OptionMenuPage.java]: It's already done.

File: src/test/java/salesforce/login/LoginTest.java (hunk 3 above)
Comment (JRojas999) [reply src/test/java/salesforce/login/LoginTest.java]: It's already done.

File: src/test/java/salesforce/assets/CreateAssetTest.java (hunk 4 above)
Comment (JRojas999) [reply src/test/java/salesforce/assets/CreateAssetTest.java]: It's already done.

**Review by edsonarios — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**